src/basic_recipes/title.jl

```diff
@@ -0,0 +1,28 @@
+@recipe(Title, titletext, plot) do scene
+    t = default_theme(scene, Text)
+    t[:align] = (:center, :bottom)
```

Comment:
You need a Theme here, something like 
```julia
@recipe(Title, titletext, plot) do scene
    Theme(
        default_theme(scene, Text)...
        align = (:center, :bottom),
    )
```
might work.

Reply:
But your way of specification looks more elegant, although they seem equivalent to me? Or am I mutating some global `Text` theme here?

Reply:
I inserted an `@show objectid(t)` and it is a new theme each time. I actually prefer my approach now. The reason is, that with your approach the `align` keyword is passed twice to `Theme`. One time  as part of `default_theme(...)` and one time explicitly. 
Now if you pass a keyword multiple times, only the last time counts in julia AFAICT. 
```julia
f(;kw...) = kw
kw = [:a=>2, :a=>1, :a=>3]
f(;kw...)
# pairs(::NamedTuple) with 1 entry:
#  :a => 3
```

But this looks like a fragile detail to me, which I don't want to rely on. The mutating approach is looks more sane to me.

Reply:
Makes sense!